### PR TITLE
Fixed the parent POM to jnosql-parent 1.0.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by dearrudam**
Ref: https://github.com/eclipse/jnosql/issues/308

### CODE REVIEW

**Comment by dearrudam** [comment]
@otaviojava, could you review this PR?!

**Comment by genie-jnosql** [comment]
Can one of the admins verify this patch?

File: pom.xml [hunk 1]
```diff
@@ -19,24 +19,67 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.eclipse.jnosql</groupId>
-        <artifactId>jnosql-parent</artifactId>
-        <version>1.0.0-b6-SNAPSHOT</version>
+        <groupId>org.eclipse.ee4j</groupId>
```
Comment (otaviojava) [inline pom.xml:22]: @dearrudam, in this case, it is acceptable to use the JNoSQL parent once both belong to the same project: Eclipse JNoSQL.
So, please keep as jnosql

File: pom.xml [hunk 2]
```diff
@@ -19,24 +19,67 @@
...
-        <connection>scm:git:git://github.com/eclipse/jnosql-diana-driver.git</connection>
-        <developerConnection>scm:git:ssh://github.com:eclipse/jnosql-diana-driver.git</developerConnection>
-        <url>https://github.com/eclipse/jnosql-diana-driver</url>
+        <connection>scm:git:git://github.com/eclipse/jnosql-communication-driver.git</connection>
+        <developerConnection>scm:git:ssh://github.com:eclipse/jnosql-communication-driver.git</developerConnection>
+        <url>https://github.com/eclipse/jnosql-communication-driver</url>
     </scm>
 
     <properties>
+        <apache.rat.version>0.13</apache.rat.version>
```
Comment (otaviojava) [inline pom.xml:52]: @dearrudam once both are JNoSQL I belive it is fine to use the same JNoSQL parent and don't duplicate those lines.

File: pom.xml (hunk 1 above)
Comment (dearrudam) [reply pom.xml:22]: @otaviojava oh, you're right! I could change the parent direct to:
```xml
   <parent>
        <groupId>org.eclipse.jnosql.mapping</groupId>
        <artifactId>jnosql-mapping-parent</artifactId>
        <version>1.0.0-SNAPSHOT</version>
    </parent>
```
Am I right?

File: pom.xml (hunk 2 above)
Comment (dearrudam) [reply pom.xml:52]: @otaviojava yeah, you're right! I'm working on it! Thanks!

File: pom.xml (hunk 1 above)
Comment (dearrudam) [reply pom.xml:22]: @otaviojava, thanks for notifying me! I'm fixing it!

File: pom.xml (hunk 2 above)
Comment (otaviojava) [reply pom.xml:52]: @dearrudam please, let me know when you finished.

Comment (dearrudam) [reply pom.xml]: @otaviojava please, go ahead!!

**Comment by dearrudam** [comment]
@otaviojava, I'd like to say that the checks ("Java 11 CI with Maven / build" and "Java 17 CI with Maven / build") are not successful because the updated jnosql 1.0.0-SNAPHOT version wasn't deployed to snapshot repositories; Once they are deployed already, both ones will get be solved.

**Comment by otaviojava** [comment]
@dearrudam ok, I'll deploy the snapshot and run the tests again.
If you wish, you can go to the extension mapping.

**Comment by otaviojava** [comment]
@dearrudam merged, thanks

**Pull request merged**